.env

```diff
@@ -162,4 +162,5 @@ SENTRY_AUTH_TOKEN=
 # Whether GA4 sends data or not. NOTE: must be set in build environment.
 NEXT_PUBLIC_GA4_DEBUG_MODE=true
 
-CURRENT_COUPON_CODE_ID=
\ No newline at end of file
+CURRENT_COUPON_CODE_ID=
+GA4_API_SECRET=unsafe-default-secret-for-dev
```

Comment:
@Vinnl your point about not setting this so we throw an error in production if it's unset makes sense... however that'll make the test fail when running locally or in CI (unless it's set to some known value).

What do we usually do in this case? I could leave it unset but handle it different in `test` and `local` environments... that is making the code-under-test behave a bit differently based on environment which complicates things but might make sense in this case.

Reply:
Hmm, I think if it really doesn't matter what this value is set to, I suppose I'd look into not making the tests fail if it's not set? i.e. I assume that if the tests actually verify GA behaviour, then this would have to be set to a true value anyway?

Agreed that modifying the code-under-test is probably no better than setting this by default.